content/rest/channels.textile

```diff
@@ -281,7 +281,7 @@ bq(definition#publish-msg-array).
   objc,swift: publish(messages: [ "ARTMessage":#message ], callback: (("ARTErrorInfo":/realtime/types#error-info?) -> Void)?)
   go:       (c &ast;RestChannel) PublishAll(messages []&ast;proto.Message) (error "ErrorInfo":#error-info)
 
-Publish several messages on this channel. <span lang="jsall,java,objc,swift">A <span lang="default">callback</span><span lang="java">listener</span> may optionally be passed in to this call to be notified of success <span lang="default">or failure</span><span lang="ruby"></span> of the operation.</span> It is worth noting that idempotency is "not fully supported":https://support.ably.io/solution/articles/3000088357-how-can-i-publish-multiple-messages-in-a-single-rest-request-with-idempotency when sending serveral messages at once with this method.
+Publish several messages on this channel. <span lang="jsall,java,objc,swift">A <span lang="default">callback</span><span lang="java">listener</span> may optionally be passed in to this call to be notified of success <span lang="default">or failure</span><span lang="ruby"></span> of the operation.</span> "It is worth noting that there are additional considerations and constraints if you want to publish multiple messages idempotently in one publish operation with client-supplied IDs.":https://support.ably.io/solution/articles/3000088357-how-can-i-publish-multiple-messages-in-a-single-rest-request-with-idempotency
```

Comment:
Not major, but "It is worth noting that" is not really part of the link i.e. if you think about what the link content is, it is "there are additional considerations and constraints if you want to publish multiple messages idempotently in one publish operation with client-supplied IDs".